src/StatPlots.jl

```diff
@@ -2,8 +2,8 @@
 module StatPlots
 
 using Reexport
-@reexport using Plots
-import Plots: Plot, cycle
+using RecipesBase, Plots
```

Comment:
>And why do you need to import RecipesBase?

Because I'm dumb. I thought I backtracked everything before squashing. Looks like I missed one of the changes.